index.html

```diff
@@ -864,9 +864,12 @@
 </aside>
 
 A [=person=]'s <dfn>identity</dfn> is the set of characteristics that define them. Their identity
-*in a [=context=]* is the set of characteristics they present to that context. People
-frequently present different identities to different contexts, and also frequently share an identity
-among several contexts.
+*in a [=context=]* is the set of characteristics they present to that context,
+directly or indirectly. People frequently present different identities to
```

Comment:
"directly or indirectly" shows up in a couple places, and I'm never sure exactly what it means. Here, is it
```suggestion
*in a [=context=]* is the set of characteristics they present to that context,
or that the context can infer. People frequently present different identities to
```
?